test/test_bandedlu.jl

```diff
@@ -24,11 +24,7 @@ let
     @test BandedMatrices._promote_to_blas_type(typ, Int16)      == typ
     @test BandedMatrices._promote_to_blas_type(typ, Int8)       == typ
 
-    if VERSION < v"0.5.0-dev+5122"
```

Comment:
Since git has a historical backup, it doesn't make sense to pollute the current version of the package with outdated work arounds.  I think ideally a package should only contain code for currently supported versions of Julia, which excludes old development versions.

So I'm in favour of removing the special case completely.